ios/Classes/FacebookModule.m

```diff
@@ -608,12 +622,29 @@ -(void)presentMessengerDialog:(id)args
    
     TiThreadPerformOnMainThread(^{
         FBSDKShareLinkContent *content = [[FBSDKShareLinkContent alloc] init];
+        
+        NSString *description = [params objectForKey:@"description"];
+        NSString *title = [params objectForKey:@"title"];
+        NSString *picture = [params objectForKey:@"picture"];
+        
+        if (description != nil) {
+            NSLog(@"Ti.Facebook.presentMessengerDialog.description has been deprecated in Ti.Facebook 5.5.0 as of the Graph v2.9 changes.");
+            content.contentDescription = [params objectForKey:@"description"];
```

Comment:
As per FBSDKShareLinkContent.h - 
@property (nonatomic, readonly) NSString *contentDescription
  DEPRECATED_MSG_ATTRIBUTE("`contentDescription` is deprecated from Graph API 2.9");

contentDescription  has changed to read only, but you are assigning it. It is giving build error. Please remove it.